### ISSUE TITLE
Security Infrastructure (D4.1): Detail Authorization Use Cases for the Bioinformatics Use Case

### ISSUE THREAD

**Issue by mathiasslawik** [cyclone-project/cyclone-tracking#109]
To design the security infrastructure I need more information about the authorization aspect of the bioinformatics use case. This relates to #105 and #70.

This includes:
* Which systems authorize users?
* On which basis should the system authorize users (e.g., user roles, user organization, user attributes, user location, ...)
* How should the authorization rules be defined? Are there global rules or is there a need for the consumers to customize them?

The more information, the better.

**Comment by mathiasslawik** [comment]
Christophe and I talked about this subject.

If I got it correctly there are currently certain groups within the cloud portal which are used for authorizing user access. In the future, the users should be authorized based on the lab from which they authenticate (their "home realm"). There will be some VMs available for some labs and not for others.

I currently do not see the need to have XACML authorization for this purpose.

Christophe will extend the use case description by this information, if it's not yet included.
